v1/ao/layer.go

```diff
@@ -133,14 +134,21 @@ type Profile interface {
 	Err(error)
 }
 
+type ErrorOptions struct {
+	Timestamp time.Time
+	Class     string
+	Msg       string
+}
+
 // SpanOptions defines the options of creating a span
 type SpanOptions struct {
 	// WithBackTrace indicates whether to include the backtrace in BeginSpan
 	// Keep in mind that the cost of this option may be high as it calls
 	// `debug.Stack()` internally to gather the stack trace. Please consider
 	// the impact on performance/memory footprint carefully.
 	WithBackTrace bool
-
+	StartTime     time.Time
+	EndTime       time.Time
```

Comment:
is `EndTime` being used? :)

Reply:
It's not used in the OT tracer, the Timestamp in OT's SpanConfig is used as timestamp override instead.